# Essay list about Molecular Generation or Drug Discovery 
[![Build Status](https://travis-ci.com/agitter/manubot-awesome-list.svg?branch=master)](https://travis-ci.com/agitter/manubot-awesome-list)

Inspired by YanZhe Zhang's [papers_for_protein_design_using_DL](https://github.com/Peldom/papers_for_protein_design_using_DL), I have a tendency to organize drug discovery papers by deep learning published in recent years especially on molecular generation, and this repo in the future will always be dynamic.We will make this list by [Manubot](http://manubot.org), If you know some literature in this regard, I also very welcome you to put forward the doi/url/arxiv/PMID/ and so on of the literature collected in this issue in the issue, On the other way, you can also contribute by create or edit the file in the content directory, as follows is for example: 
```ruby
## Manubot example documention and introduction link
url:https://greenelab.github.io/meta-review/ 
doi:10.1098/rsif.2017.0387 
url:https://github.com/manubot/manubot/ 
``` 
In this repository, `README.md` is created via continuous integration and should not be edited directly.
Edit `README-BASE.md` to update this text.
Update the reference lists in the `content` directory to add new sections or references.
This is only a proof of concept that is not robust against errors in the scripts or merge conflicts.

The `.travis.yml`, `deploy.sh`, and `environment.yml` files were derived from <https://github.com/manubot/rootstock> (CC0 1.0 license).

This repository also contains a GitHub Actions workflow that uses Manubot to automatically extract reference information from the identifier in an issue title.
The workflow only runs on issues with the label `reference`.
See [#7](https://github.com/agitter/manubot-awesome-list/issues/7) for an example.
## Manubot
**Open collaborative writing with Manubot**   
Daniel S. Himmelstein, Vincent Rubinetti, David R. Slochower, Dongbo Hu, Venkat S. Malladi, Casey S. Greene, Anthony Gitter  
*Manubot*, May 2020, <https://greenelab.github.io/meta-review/>

**Opportunities and obstacles for deep learning in biology and medicine**   
Travers Ching, Daniel S. Himmelstein, Brett K. Beaulieu-Jones, Alexandr A. Kalinin, Brian T. Do, Gregory P. Way, Enrico Ferrero, Paul-Michael Agapow, Michael Zietz, Michael M. Hoffman, … Casey S. Greene  
*Journal of The Royal Society Interface*, April 2018, <https://doi.org/gddkhn>   
DOI: [10.1098/rsif.2017.0387](https://doi.org/10.1098/rsif.2017.0387) · PMID: [29618526](https://www.ncbi.nlm.nih.gov/pubmed/29618526) · PMCID: [PMC5938574](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5938574)

**GitHub - manubot/manubot: Python utilities for Manubot: Manuscripts, open and automated**   
GitHub  
<https://github.com/manubot/manubot>


## Protein-engineering


## Single-cell-pseudotime
**A comparison of single-cell trajectory inference methods**   
Wouter Saelens, Robrecht Cannoodt, Helena Todorov, Yvan Saeys  
*Nature Biotechnology*, April 2019, <https://doi.org/gfxsgd>   
DOI: [10.1038/s41587-019-0071-9](https://doi.org/10.1038/s41587-019-0071-9) · PMID: [30936559](https://www.ncbi.nlm.nih.gov/pubmed/30936559)

**GitHub - agitter/single-cell-pseudotime: An overview of algorithms for estimating pseudotime in single-cell RNA-seq data**   
GitHub  
<https://github.com/agitter/single-cell-pseudotime>

**Network Inference with Granger Causality Ensembles on Single-Cell Transcriptomic Data**   
Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter  
*Cold Spring Harbor Laboratory*, January 2019, <https://doi.org/gft4bb>   
DOI: [10.1101/534834](https://doi.org/10.1101/534834)


